### PR TITLE
Ready Run Task Docs for GA

### PR DESCRIPTION
**Pull request by ritsok**
### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages.
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 1]
```diff
@@ -93,18 +91,23 @@ To get started, [create an HCP Packer account](https://cloud.hashicorp.com/produ
...
 
 ### Bridgecrew
 
 [Bridgecrew](https://bridgecrew.io/) helps teams address security and compliance errors in Terraform as part of each and every code review.
 
 To get started, sign up for an eligible [pricing](https://bridgecrew.io/pricing/) plan, and follow the instructions in the [Integration via Run Tasks](https://docs.bridgecrew.io/docs/integrate-with-terraform-cloud#integration-via-run-tasks) user documentation.
 
+### Firefly
+
+Firefly integrates with Terraform Cloud to help you move faster without breaking your cloud. The run tasks integration helps teams cast a cloud safety net and understand how changing untargeted assets with affect your cloud before you deploy it.
```
Comment (laurapacilio) [inline content/cloud-docs/integrations/run-tasks/index.mdx:107]:
```suggestion
Firefly integrates with Terraform Cloud to help you move faster without breaking your cloud. The run tasks integration helps teams cast a cloud safety net and understand how changing untargeted assets will affect your cloud before you deploy it.
```

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 2]
```diff
@@ -132,3 +135,24 @@ To get started, [sign up](https://www.lightlytics.com/signup) for a free trial,
...
+
+### Sophos
+
+Sophos Factory works with Terraform Cloud run tasks to enable IaC security tools including OPA, BridgeCrew’s Checkov, Accurics, TerraScan through a public solution catalog of prebuilt DevSecOps pipelines. To get started, follow the [Integrate with Terraform Cloud](https://docs.refactr.it/docs/tutorials/terraform-cloud/) tutorial in the Sophos documentation.
+
+### Tenable
+
+Tenable.cs and Hashicorp Terraform Cloud enables teams to codify cloud security policies for infrastructure as code with automated enforcement and remediation recommendations as part of their development workflow. For Terraform Cloud users, it has never been easier to establish compliance and security guardrails in the development process that ensures infrastructure is secure before it is provisioned.To get started, follow the [Configure Terraform Cloud Integration guide](https://docs.tenable.com/tenablecs/Content/Quickstart/CI-CD/Terraform.htm) in the Tenable documentation.
```
Comment (laurapacilio) [inline content/cloud-docs/integrations/run-tasks/index.mdx:153]:
```suggestion
Tenable.cs and Hashicorp Terraform Cloud enables teams to codify cloud security policies for infrastructure as code with automated enforcement and remediation recommendations as part of their development workflow. This lets you establish compliance and security guardrails in the development process that ensure infrastructure is secure before it is provisioned. To get started, follow the [Configure Terraform Cloud Integration guide](https://docs.tenable.com/tenablecs/Content/Quickstart/CI-CD/Terraform.htm) in the Tenable documentation.
```

**Review by laurapacilio — APPROVED** [review]

File: content/cloud-docs/workspaces/settings/run-tasks.mdx [hunk 3]
```diff
@@ -4,12 +4,12 @@ page_title: Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise
 
 # Run Tasks
 
--> **Note:** As of September 2021, Run Tasks are a private beta feature, subject to change, and not all customers will see this functionality in their Terraform Cloud organization.
-
 Run Tasks allow you to directly integrate third-party tools and services at certain stages in the Terraform Cloud run lifecycle. When triggered, a run task sends an API payload to the external service. This payload contains a collection of run-related information, including a callback URL that the service uses to respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the run task's enforcement settings that have been configured for the workspace.
 
 You can manage run tasks through the UI as detailed below or through the [Run Tasks APIs](/cloud-docs/api-docs/run-tasks).
 
+> **Hands-on:** Try the [Configure Snyk Run Task in Terraform Cloud tutorial](https://learn.hashicorp.com/tutorials/terraform/cloud-run-tasks-snyk&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) on HashiCorp Learn.
```
Comment (ritsok) [inline content/cloud-docs/workspaces/settings/run-tasks.mdx:11]:
```suggestion
> **Hands-on:** Try the [Configure Snyk Run Task in Terraform Cloud tutorial](https://learn.hashicorp.com/tutorials/terraform/cloud-run-tasks-snyk?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) on HashiCorp Learn.
```

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 4]
```diff
@@ -132,3 +135,24 @@ To get started, [sign up](https://www.lightlytics.com/signup) for a free trial,
...
 
 To get started, request early access at [Moderne.io](https://moderne.io/), and follow the instructions in the [Terraform Cloud integration](https://docs.moderne.io/how-to/terraform-cloud-integration) user documentation.
+
+### Snyk
+
+The [Snyk](https://snyk.io/) integration for Terraform Cloud allows teams using Terraform to find, track, and fix security misconfigurations in their cloud infrastructure as part of their SDLC before they ever reach production.
+
+To get started, [create a free Snyk account](https://snyk.io/) and follow the instructions in the [Integrating Snyk with Terraform Cloud](https://docs.snyk.io/features/integrations/ci-cd-integrations/integrating-snyk-with-terraform-cloud) user documentation.
+
+> **Hands-on:** Try the [Configure Snyk Run Task in Terraform Cloud tutorial](https://learn.hashicorp.com/tutorials/terraform/cloud-run-tasks-snyk&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) on HashiCorp Learn.
```
Comment (ritsok) [inline content/cloud-docs/integrations/run-tasks/index.mdx:145]:
```suggestion
> **Hands-on:** Try the [Configure Snyk Run Task in Terraform Cloud tutorial](https://learn.hashicorp.com/tutorials/terraform/cloud-run-tasks-snyk?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) on HashiCorp Learn.
```

**Review by cam-stitt — APPROVED** [review]
Looks great!

**Pull request merged**